client/app/components/PlainButton.tsx

```diff
@@ -0,0 +1,20 @@
+import classNames from "classnames";
+import React from "react";
+
+import "./PlainButton.less";
+
+interface PlainButtonType extends Omit<React.ButtonHTMLAttributes<HTMLButtonElement>, "type"> {
+  type?: "link" | "button";
+}
+
+function PlainButton({ className, type, ...props }: PlainButtonType) {
+  return (
+    <button
```

Comment:
What's the motivation for using this instead of an `ant` Button? How do you plan to help developers choose this instead of an `ant` Button when it's the right choice?